banhmi/prisma/schema.prisma

```diff
@@ -32,18 +32,21 @@ model Space {
 }
 
 model User {
-  id          Int             @id @default(autoincrement())
-  username    String          @unique
-  space       Space?          @relation("space", fields: [spaceId], references: [id])
-  spaceId     Int?            @map("space_id")
-  friend1     Friendship[]    @relation("friend1")
-  friend2     Friendship[]    @relation("friend2")
-  sendTo      FriendRequest[] @relation("send")
-  receiveFrom FriendRequest[] @relation("receive")
-  categories  UserCategory[]  @relation("category")
-  active      Boolean         @default(true)
-  createdAt   DateTime        @default(now()) @map("created_at")
-  updatedAt   DateTime        @updatedAt @map("updated_at")
+  id            Int             @id @default(autoincrement())
+  username      String          @unique
+  space         Space?          @relation("space", fields: [spaceId], references: [id])
+  spaceId       Int?            @map("space_id")
+  friend1       Friendship[]    @relation("friend1")
+  friend2       Friendship[]    @relation("friend2")
+  sendTo        FriendRequest[] @relation("send")
+  receiveFrom   FriendRequest[] @relation("receive")
+  categories    UserCategory[]  @relation("category")
+  active        Boolean         @default(true)
+  createdAt     DateTime        @default(now()) @map("created_at")
+  updatedAt     DateTime        @updatedAt @map("updated_at")
+  authToken     AuthToken?
+  salt          String
+  hashPassword  String
```

Comment:
I think we can use @map here, which converts the column name to snake case in the database.